src/main/java/org/pac4j/dropwizard/J2EHelper.java

```diff
@@ -32,7 +28,15 @@ private J2EHelper() {
     public static void registerSecurityFilter(Environment environment,
             Config config, ServletSecurityFilterConfiguration fConf) {
 
-        final SecurityFilter filter = new SecurityFilter();
+        final SecurityFilter filter = new SecurityFilter(config);
+        /*	Merely passing in config to the ctor above doesn't apply any security
+        	logic specified in the config. Seems like a bug in the SecurityFilter
+        	to me. Attempt to work around it by doing that work myself if I
+        	have any security logic set in the config.
```

Comment:
I would prefer that you made a PR to jax-rs-pac4j ;)

Reply:
sorry, I meant j2e-pac4j

Reply:
I was asked to report this and provide a fix PR in the dropwizard-pac4j project by Jérôme:
https://groups.google.com/forum/#!topic/pac4j-users/u6WecNrIN9I

I understand this straddles other parts of pac4j, but since there seem to be some concern around my PR by you (victornoel) above, perhaps it's better that someone more knowledgable figures out the proper fix instead of just merging in my PR (I'm totally new to pac4j, and was just happy to make this work in my ue case).

-JM

Reply:
I understand your position yes, but this seems to be in the end a bug in j2e-pac4j, not dropwizard-pac4j, so it is useless to fix it here. @leleuj can you confirm in case I missed something?

Reply:
The problem here is that even if you define a specific `securityLogic` in the YAML file in Dropwizard, it is never taken into account: the `SecurityFilter` is created without checking if a new `securityLogic` has been provided in the configuration.

Does it make sense?

Reply:
@leleuj my understanding was that the `SecurityFilter` implementation from `j2e-pac4j` should use the `SecurityLogic` of the `Config` instead of its own.

Reply:
Ah OK. I see. Indeed, it could be solved in `j2e-pac4j`. It's not really a bug, more a capability which has been put into action. I opened this issue: https://github.com/pac4j/j2e-pac4j/issues/48

Reply:
That said, there is still the question about to set the `profileStorageDecision` in Dropwizard...

Reply:
OK. It's fixed in j2e-pac4j v4.1.0-SNAPSHOT: @TheWizz would you mind giving it a try? Thx

Reply:
Yes, j2e-pac4j v4.1.0 solved my problem. Many thanks!
-JM